WORKSPACE

```diff
@@ -248,7 +248,12 @@ register_jq_toolchains()
 load("@vaticle_dependencies//tool/common:deps.bzl", vaticle_dependencies_tool_maven_artifacts = "maven_artifacts")
 load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_maven_artifacts = "artifacts")
 load("@vaticle_typeql//dependencies/maven:artifacts.bzl", vaticle_typeql_maven_artifacts = "artifacts")
-load("//dependencies/maven:artifacts.bzl", vaticle_typedb_driver_java_maven_artifacts = "artifacts", vaticle_typedb_driver_java_maven_overrides = "overrides")
+load(
+    "//dependencies/maven:artifacts.bzl",
+    vaticle_typedb_driver_java_maven_artifacts = "artifacts",
+    vaticle_typedb_driver_java_maven_overrides = "overrides",
+    vaticle_typedb_driver_java_internal_artifacts = "internal_artifacts",
```

Comment:
`vaticle_typedb_driver_internal_maven_artifacts` sorry yes this is a long name. Although we can drop 'java' here since 'maven' implies java! You can apply it to all the `typedb_driver_java_maven...` in this file